migrate/docker/migrate.sh

```diff
@@ -67,27 +67,27 @@ if (( $PSQLVERSION < $PSQLVERSION_DOCKER )); then
 
 	echo "Migrating old cluster"
 	{
-		pg_upgradecluster -m upgrade $PSQLVERSION main
+		pg_upgradecluster -m upgrade "$PSQLVERSION" main
 	} ||
 	{
 		# could also check psotgresql owner
 		#USER=$(stat -c '%U' "/etc/postgresql/12/main")
 		#echo $USER
 		#USER=$(stat -c '%U' "/var/lib/postgresql/12/main")
 		#echo $USER
-		chown -R postgres:postgres $PG_CFG
-		chown -R postgres:postgres $PG_DATA
-		pg_upgradecluster -m upgrade $PSQLVERSION main
+		chown -R postgres:postgres "$PG_CFG"
+		chown -R postgres:postgres "$PG_DATA"
+		pg_upgradecluster -m upgrade "$PSQLVERSION" main
 	}	
 fi
 
 echo "making new folders"
-mkdir $DIR 
+mkdir "$DIR"
 
 echo "Copying docker-compose file"
-cp docker-compose.yml $DIR
+cp docker-compose.yml "$DIR"
 
-cd $DIR
+cd "$DIR" || exit 0
```

Comment:
You could make it cleaner by using msg_box here, stating why it exists.

if ! cd "$DIR"
then
    msg_box "$DIR was not found, this script will exit. Please report this to $ISSUES.
    exit 1
fi

In general, try to be more verbose on why things happen so that the user knows what's going on instead of the script just exiting.

Thanks!